src/main/resources/liquibase/harness/change/expectedSql/edb/createView.sql

```diff
@@ -0,0 +1,3 @@
+INVALID TEST
+-- This does not appear to be a genuine failure, the test is failing because the expected snapshot file is not found when it in fact is present
+-- This is needs to be investigated further by Oleh
```

Comment:
@KushnirykOleh -- this test is failing because the expected snapshot json is not being detected, when does exist. It needs to be looked at.

Reply:
We don't have default expected snapshot (createView.json) file in root folder. They are different for different platforms. I'll add some, even it will fail for new platforms, but it will be less confusing

Reply:
@KushnirykOleh , In the expected Snapshot folder for edb, I put the `createView.json` file in there, but still it failed because the expectedSnapshot was being reported as null. Should it not have picked that up? It did for the other 2 snapshots I added for null constraints. But for this view it did not work the same.